### PR TITLE
ROC-3148 show claim details

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3148

### Change description
Add a view claim details page linked off of the dashboard

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [x] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/common/interest.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { InterestDateType } from 'app/common/interestDateType'
+import { MomentFactory } from 'common/momentFactory'
+import { Moment } from 'moment'
+import { InterestType } from 'claim/form/models/interest'
+import { calculateInterest } from 'app/common/calculateInterest'
+import { Claim } from 'claims/models/claim'
+
+export async function getInterestDetails (claim: Claim): Promise<object> {
```
Comment (timja-kainos) [inline src/main/common/interest.ts]: what is this? p.s. don't return an object an actual type

File: src/main/features/response/routes/claim-details.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { Claim } from 'claims/models/claim'
+import { getInterestDetails } from 'common/interest'
+import { MomentFactory } from 'common/momentFactory'
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.claimDetailsPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    try {
+      const claim: Claim = res.locals.claim
+      const feeAmount: number = claim.claimData.feeAmountInPennies / 100
```
Comment (timja-kainos) [inline src/main/features/response/routes/claim-details.ts]: convert pennies to pounds class

File: src/main/features/response/routes/task-list.ts [hunk 3]
```diff
@@ -28,7 +28,8 @@ export default express.Router()
           submitSection: submitSection,
           respondToClaimSection: respondToClaimSection,
           claim: claim,
-          isAfter4pm: isAfter4pm()
+          isAfter4pm: isAfter4pm(),
+          paths: Paths
```
Comment (timja-kainos) [inline src/main/features/response/routes/task-list.ts]: preferred approach is to add these as nunjucks globals see nunjucks/index.ts

File: src/main/features/response/views/claim-details.njk [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            {% if claim.claimData.interest.rate %}
+              <tr>
+                <td scope="row">
+                  <details role="group">
+                    <summary role="button" aria-controls="details-content-0" aria-expanded="false">
+                      {{ t('Interest') }}
+                    </summary>
+                    <div class="panel text-small" id="details-content-0" aria-hidden>
+                      <div class="">
+                        {{ t('Interest calculated at %s%% for %s days', {
```
Comment (timja-kainos) [inline src/main/features/response/views/claim-details.njk]: should this be a macro as its used in more than one place?

File: src/main/features/response/routes/task-list.ts (hunk 3 above)
Comment (robertparkinson) [reply src/main/features/response/routes/task-list.ts]: done

**Pull request merged**